vcluster/_fragments/sleepmode-istio-example.mdx

```diff
@@ -0,0 +1,112 @@
+import Flow, { Step } from '@site/src/components/Flow';
+import CodeBlock from '@theme/CodeBlock'
+import SleepmodeIstioConfig from '!!raw-loader!@site/vcluster/configure/vcluster-yaml/_code/sleepmode-istio-config.yaml'
+import SleepmodeIstioExample from '!!raw-loader!@site/vcluster/configure/vcluster-yaml/experimental/_code/sleepmode-istio-examples.yaml'
+
+:::info
+This example picks up where the [Istio Integration](../vcluster-yaml/integrations/istio) example leaves off.
+:::
+
+<Flow id="sleepmode-ingress-example">
+  <Step>
+    Complete the [Istio Integration](../vcluster-yaml/integrations/istio) example.
+  </Step>
+
+  <Step>
+    Install the Istio ingress gateway controller to allow routing from outside the cluster.
+
+    Save the following manifest as `ingress.yaml`:
+
+    ```yaml
+    apiVersion: install.istio.io/v1alpha1
+    kind: IstioOperator
+    metadata:
+      name: ingress
+    spec:
+      profile: empty # Do not install CRDs or the control plane
+      components:
+        ingressGateways:
+        - name: istio-ingressgateway
+          namespace: istio-ingress
+          enabled: true
+          label:
+            istio: ingressgateway
+      values:
+        gateways:
+          istio-ingressgateway:
+            injectionTemplate: gateway
+    ```
+    </Step>
+
+    <Step>
+    Create the `istio-ingress` namespace and install the ingress gateway:
+   
+    ```shell
+    kubectl create namespace istio-ingress                                                                                                      
+    istioctl install -f ingress.yaml
+    ```
+    </Step>
+    <Step>
+    Create a `Gateway` resource that uses a selector matching the newly installed ingress gateway in your vCluster.
+
+    ```yaml
+    apiVersion: networking.istio.io/v1
+    kind: Gateway
+    metadata:
+      name: istio-sm-gateway
+      namespace: test
+    spec:
+      # The selector matches the ingress gateway pod labels.
+      # If you installed Istio using Helm following the documentation, this is "istio=ingress"
+      selector:
+        istio: ingressgateway
+      servers:
+      - port:
+          number: 80
+          name: http
+          protocol: HTTP
+        hosts:
+        - "smdemo.local"
+    
+    ```
+  </Step>
+
+  <Step>
+    Update the vCluster with sleepMode enabled.
+
+    Use the following `vcluster.yaml` to update your virtual cluster on your host cluster with `--upgrade`.
+    Save this file as `vcluster.yaml`
+
+    <CodeBlock title="vCluster config for auto sleep" language="yaml">{SleepmodeIstioConfig}</CodeBlock>
+   
+    Run the following command:
+
+    ```bash title="Create vCluster with autoSleep config"
+    vcluster create <your-vcluster-name> -f vcluster.yaml --upgrade
+    ```
+  </Step>
+  
+  <Step>
+    Enable local DNS resolution for the virtual cluster.
```

Comment:
The idea of moving away from kind was less about kind itself being the problem, and more about using a non-local or real cluster. Like an EKS cluster or something in the cloud.